packages/commonwealth/client/scripts/views/pages/CreateCommunity/steps/BasicInformationStep/BasicInformationForm/constants.ts

```diff
@@ -1,30 +1,39 @@
+import NodeInfo from 'models/NodeInfo';
 import app from 'state';
 
-export const POLOGON_CHAIN_OPTION = {
-  label: 'Polygon',
-  value: 'polygon',
-};
+// used for default chain dropdown options
+export const POLYGON_ETH_CHAIN_ID = 137;
+export const ETHEREUM_MAINNET_ID = '1';
+export const OSMOSIS_ID = 'osmosis';
```

Comment:
wouldn't the ids change if ever the db changes?

Reply:
those are not our ids, those are world recognised ids so they can't really change